initfiles/sbin/hpcc_setenv.in

```diff
@@ -23,12 +23,15 @@
 ###<REPLACE>###
 
 function kill_process () {
-    SENTINEL=$1
-    PID=$2
-    TIMEOUT=$3
-    if [ -e $SENTINEL ]; then
-        rm -f $SENTINEL
+    if [ $# -eq 3 ]; then
+        SENTINEL=$1
+        shift
+        if [ -e $SENTINEL ]; then
+            rm -f $SENTINEL
+        fi
     fi
+    PID=$1
+    TIMEOUT=$2
```

Comment:
I did that so I wouldn't break the code in the other 11 init files where it's called.  But I'm going to go through and swap around the parameters, and add the change you talked about above for the process name check.